README.md

```diff
@@ -41,7 +41,12 @@
 
 [QuickRecall](https://github.com/kaiwalyakoparkar/Data-Structures-And-Algorithm-Interview-Kit/tree/main/QuickRecall)
 
+***
+<h1 align="center">Made with ❤️ by AWESOME CONTRIBUTORS </h1>
```

Comment:
Please convert the text into lowercase. Also, you don't need to create a new PR. Pushing changes to same branch will update the PR as well :)

Reply:
Okay will keep in mind from next time!